src/main/scala/info/bethard/anafora/Anafora.scala

```diff
@@ -42,6 +43,26 @@ class Entity(xml: Elem) extends Annotation(xml) {
   def text(implicit data: Data): String = spans.map{
     case (start, end) => data.text.substring(start, end)
   }.mkString("...")
+  private def recursiveSpan(entity: Entity, start: Int, end: Int)(implicit data:Data): (Int, Int) = {
```

Comment:
These changes don't seem to belong in this pull request; as far as I can tell, AnaforaReader doesn't use any of these methods.